io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/Config.java

```diff
@@ -0,0 +1,39 @@
+package io.openems.edge.goodwe.et;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition( //
+		name = "Goodwe ET Battery-Inverter", //
+		description = "Implements the Goodwe battery inverter for 3 phase systems.")
+
+@interface Config {
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "ess0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
+
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+	
+	@AttributeDefinition(name = "Modbus Unit-id", description = "Unit-id")
+	int unit_id() default 1;
```

Comment:
Is it really "1" by default. I remember from the Docs that it was another number.